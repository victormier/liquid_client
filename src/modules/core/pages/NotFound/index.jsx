import React from 'react';
import { Link } from 'react-router';
import { Grid } from 'react-bootstrap';


const NotFound = () => (
  <Grid>
    <h2>Page not found</h2>
    <Link to="/">
      <span className="glyphicon glyphicon-home" /> Go back home
    </Link>
  </Grid>
);

export default NotFound;
