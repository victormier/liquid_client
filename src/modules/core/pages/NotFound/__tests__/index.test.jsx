import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import NotFound from '../index'


describe('component', () => {
  it('renders', () => {
    const container = shallow(
      <NotFound />
    )

    expect(shallowToJson(container)).toMatchSnapshot()
  })
})
