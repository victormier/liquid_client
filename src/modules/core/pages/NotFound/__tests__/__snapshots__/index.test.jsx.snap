// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <h2>
    Page not found
  </h2>
  <Link
    onlyActiveOnIndex={false}
    style={Object {}}
    to="/"
  >
    <span
      className="glyphicon glyphicon-home"
    />
     Go back home
  </Link>
</Grid>
`;
